src/index.html

```diff
@@ -1,13 +1,162 @@
 <!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Dia</title>
-    <link rel="stylesheet" href="./styles/main.scss">
-  </head>
-  <body>
-    <h1>Dia</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
-  </body>
+<html lang="en" class="page">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Dia</title>
+  <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;600&family=Poppins:wght@600;700&display=swap"
+    rel="stylesheet">
+  <link rel="stylesheet" href="./styles/main.scss">
+</head>
+
+<body class="page__container">
+  <header class="header">
+    <div class="container">
+      <div class="header__top">
+        <div class="header__logo-box">
+          <div class="header__logo"></div>
+          <div class="header__burger"></div>
+        </div>
+        
+        <div class="header__nav">
+          <ul class="header__list">
+            <li class="header__item">
+              <a href="#about-us" class="header__link">About us</a>
+            </li>
```

Comment:
Пам'ятай про стиль написання коду, перевір це по всій сторінці
```suggestion
            <li class="header__item">
              <a href="#about-us" class="header__link">
                About us
              </a>
            </li>
```